vgn_grasp_gen3_lite:
  base_frame_id: base_link
  tool0_frame_id: panda_link8
  T_tool0_tcp: {rotation: [0.000, 0.000, 0.707, 0.707], translation: [0.000, 0.000, -0.038]}
  finger_depth: 0.05
  max_opening_width: 0.1
  depth_cam:
    frame_id: camera_depth_optical_frame
    topic_name: /camera/depth/image_rect_raw
    intrinsic:
      height: 480
      width: 640
      K: [379.3564147949219, 0.0, 323.6485290527344, 0.0, 379.3564147949219, 236.95474243164062, 0.0, 0.0, 1.0]
  scan_joints_right:
    - [-2.321415470568673, 0.1928988830249464, -1.8737681850466705, 2.397480943939625, -1.2749660013222783, -0.3798551883934902]
    - [-1.7535429734283516, 1.4039998845621438, -0.13367683828430987, 2.276991014108952, -2.2583419622595633, 2.318418082761789]
    - [-1.0895268988267537, 1.3321664404794364, -0.1954419355498631, -1.7220180703406482, 1.9340760005637334, 1.7497410446561548]
    - [-0.3077389166030162, 0.7905441895718084, -1.7893086942358982, -2.525626929272058, 1.1961843699555545, 0.9856580236753737]
    # - [-0.3266846445978402, 0.5120806038837061, -1.9263565618280296, 0.6727033060774372, -1.2374154299521614, 0.8090054885643466]
    # - [1.876153844751164, -1.4104793554943065, 0.06728609652139698, -1.1176839683999429, -2.080927230444556, 2.231484515298332]
    # - [1.1495590771709923, -1.1038664234001754, 0.0746302958595085, -0.8468926834962494, -2.3063824588473936, 2.5190617045527945]
    # - [2.0469418328257274, -0.14165566891018688, 1.4761949859208838, -1.5786920667947637, -1.5091430822951262, 0.8789872367411219]
  scan_joints_left:
    - [-0.4550958807748442, -1.0388245727293883, 1.1614623419792105, -2.1489227930803505, -2.0043797249162987, -1.7113100322296573]
    - [-1.328647872047224, -1.0714525571406037, 0.5622348514287324, -2.039847970630084, -1.9130998785538331, -1.7113435880593917]
    - [-2.1452156728429594, -1.2496904720485542, 0.2833582094555589, -1.1303451688543982, -1.8565375327938582, -1.0469291045587177]
  aligned_cam:
    frame_id: camera_color_optical_frame
    topic_name: /camera/aligned_depth_to_color/image_raw
    intrinsic:
      height: 480
      width: 640
      K: [601.7557373046875, 0.0, 322.140380859375, 0.0, 601.8693237304688, 232.57798767089844, 0.0, 0.0, 1.0]
  deposit_points:
    apple: [1.494887180979932, 0.19212017472220724, -1.6757142210996578, 1.572355873929247, -1.2755726694185983, -0.3803995385203027]
    mouse: [-2.025708449454526, 0.19212017472220724, -1.675721677950711, 1.5723521455037206, -1.276993199544047, -0.38039208166924965]
    # spoon: [-2.025708449454526, 0.19212017472220724, -1.675721677950711, 1.5723521455037206, -1.276993199544047, -0.38039208166924965]
    # knife: [-2.025708449454526, 0.19212017472220724, -1.675721677950711, 1.5723521455037206, -1.276993199544047, -0.38039208166924965]